specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineScaleSet.json

```diff
@@ -3780,6 +3780,10 @@
           "$ref": "./computeRPCommon.json#/definitions/CreateOption",
           "description": "Specifies how the virtual machines in the scale set should be created.<br><br> The only allowed value is: **FromImage** \\u2013 This value is used when you are using an image to create the virtual machine. If you are using a platform image, you also use the imageReference element described above. If you are using a marketplace image, you  also use the plan element previously described."
         },
+        "deleteOption": {
```

Comment:
please add this new property to the end of the properties list. so after "managedDisk" in this list

Reply:
done